Buildings/Controls/OBC/FDE/DOAS/DehumMode.mo

```diff
@@ -0,0 +1,150 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block DehumMode
+  "This block calculates when dehumidification mode is active."
+
+  parameter Real dehumSet(
+    final min=0,
+    final max=100)=60
+   "Dehumidification set point.";
+
+  parameter Real delayTimeDehFal(
```

Comment:
Rename to timThrDehDis. Continuous time period for which measured relative humidity needs to fall below relative humidity threshold before dehumidification mode is disabled. Same comments apply to next parameter